### ISSUE TITLE
How to adjust the size of inline math

### ISSUE THREAD

**Issue by a858438680** [marktext/marktext#2696]
### Question description
I'm trying to use Mark Text recently. There is only one thing which I found very annoying. The size of inline math is far more larger than plain text. Like this picture below:

![Screenshot 2021-11-29 134413](https://user-images.githubusercontent.com/25167196/143815034-b6364c84-5614-4696-9dc7-797f1bd2f065.png)

Is there any way to adjust the size of inline math?

My preferences are like this:

![image](https://user-images.githubusercontent.com/25167196/143815122-cedb9a49-05ed-4fd6-8eb0-f827b2504f52.png)

**Comment by Unturned3** [comment]
Yes, I find it quite annoying too that the math font is larger than the surrounding text. I'd also like to know a way to adjust this. Is there a way to access the CSS file used by KaTeX to change the font size?

**Comment by Jocs** [comment]
Thanks for the question, and sorry for the long silence!

The reason inline math looks bigger than the surrounding text is that KaTeX renders at `1.21em` by design (the math fonts are intentionally sized up a little so the x-height of symbols matches the body text). MarkText doesn't currently expose a preference to change the inline-math font size.

If the default sizing bothers you, you can override it with a custom theme: copy one of the built-in themes and add a rule such as

```css
.mu-math > .mu-math-render .katex { font-size: 1em; }
```

We're closing this as not planned since there isn't a dedicated setting and the sizing is KaTeX's intended behavior, but if you feel inline math should match the body size out of the box, please reopen with a concrete proposal (or screenshots of how you'd like it to look) and we'll reconsider. Thanks for using MarkText!
